docs/docs/guides/quick-start/gather-insights.md

```diff
@@ -13,17 +13,39 @@ First of all, you'll need to authenticate by running:
 tuist auth
 ```
 
+## Create an organization (optional)
+
+Unless you work on a personal project, you will need to create a Tuist organization, so that all your teammates get access to the new Tuist project. Run the following to create one:
+```bash
+tuist organization create MyOrganization
+```
+
+You will then need to invite your teammates:
+```bash
+tuist organization invite MyOrganization teammate1@org.com
+tuist organization invite MyOrganization teammate2@org.com
+```
+
+Alternatively, you can use Google single sign-on (SSO):
+```bash
+tuist organization update --provider google --organization-id org.com
+```
+
+That way, anyone in your Google organization automatically gets access to projects created in your Tuist organization.
+
```

Comment:
I think the quickstart guide should be focused on showing what the tool is capable of, and not that much on how to use the tool. I'd move this into the `Dashboard` section (which I'd rename to `Server`). Something along the lines of `Server > Accounts and projects`